ArduPlane/tailsitter.cpp

```diff
@@ -167,10 +167,10 @@ void QuadPlane::tailsitter_output(void)
     SRV_Channels::set_output_scaled(SRV_Channel::k_aileron, (motors->get_yaw())*-SERVO_MAX);
     SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, (motors->get_pitch())*SERVO_MAX);
     SRV_Channels::set_output_scaled(SRV_Channel::k_rudder, (motors->get_roll())*SERVO_MAX);
-    SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, motors->thrust_to_actuator(motors->get_throttle()) * 100);
 
     if (hal.util->get_soft_armed()) {
         // scale surfaces for throttle
+        SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, motors->thrust_to_actuator(motors->get_throttle()) * 100);
```

Comment:
Is this actually a fix? It could make it worse we no longer override the throttle, it will just normal plane throttle.

Reply:
@IamPete1 OK, so the short answer is "I don't know".  I was going to leave a call outside the get_soft_armed to have the same code as before 0e50023593b297d930fb2d1e468f033bbd090985 - but tridge thought stuff should've been set appropriately elsewhere.

The fundamental problem being fixed here is that under no circumstances should we spin the prop up while the plane is disarmed.  The test included here clearly shows that we're doing that - simply switching to qhover is doing that.  You know better than I do what the correct fix for that is and why :-)

Reply:
I wonder if were going to spin armed? I think the fix is OK just need to add a else to set to 0 to make sure were not letting the plane throttle seek past. This function is just about the last thing called before outputting to servos.

Reply:
Added a patch.